src/otx/core/config/explain.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Config data type objects for export method."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from otx.core.types.explain import TargetExplainGroup
+
+
+@dataclass
+class ExplainConfig:
+    """DTO for explain configuration."""
```

Comment:
I see that in other files in `config` folder we use DTO abbreviation, but is it possible to write the full version here as well?

Reply:
Added